Content.Tests/DMProject/Tests/List/ListCut.dm

```diff
@@ -0,0 +1,5 @@
+﻿/proc/RunTest()
+	var/list/A = list(1,2,3)
+	ASSERT(A.len == 3)
+	A.Cut()
+	ASSERT(A.len == 0)
```

Comment:
Pretty sure we still don't have a test that confirms cutting an _empty_ list isn't an out of bounds exception.